src/reaper_osara.cpp

```diff
@@ -869,6 +894,14 @@ void postCycleRecordMode(int command) {
 	}
 }
 
+void postChangeGlobalAutomationOverride(int command) {
+	ostringstream s;
+	s << "Global Automation Override ";
```

Comment:
Consider shortening this. Maybe we could say "Global auto override" or just "Auto override" (unless you think it needs to be clearly differentiated by saying global)?

Reply:
I've changed it to just "override"